src/main/content/_includes/header.html

```diff
@@ -33,3 +43,24 @@
         </div>
     </nav>
 </header>
+
+
+<!-- What's new Modal -->
+{% assign whatsNewDoc = site.pages | where: 'name', 'whats-new.adoc' %}
+{% for doc in whatsNewDoc %}
+<div class="modal fade" tabindex="-1" id="whatsNewModal" href="{{ doc.url }}" role="dialog"
+    aria-labelledby="whatsNewModalTitle" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered whats-new-modal" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <div id="modal-body" class="modal-body"></div>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"> <span
+                        aria-hidden="true">&times;</span> </button>
+                </div>
```

Comment:
you should use liquid to put the content of the doc in here instead of loading it via jquery